src/material/legacy-tabs/tab-nav-bar/tab-nav-bar.html

```diff
@@ -7,6 +7,7 @@
      [matRippleDisabled]="_disableScrollBefore || disableRipple"
      [class.mat-tab-header-pagination-disabled]="_disableScrollBefore"
      [disabled]="_disableScrollBefore || null"
+     [attr.aria-disabled]="_disableScrollBefore || null"
```

Comment:
These elements are `aria-hidden` and they aren't focusable so the `aria-disabled` isn't necessary.

Reply:
I have a PR coming up to remove the aria-hidden. This PR (and the removal of the aria-hidden) are part of a accessibility audit for GCP.

Reply:
1. I don't think that removing the `aria-hidden` is the right decision, because all it does is change the scroll position. Users can do that with the keyboard by navigating to the next tab.
2. Setting `aria-disabled` here doesn't have an effect, because the element is already a native `button` on which we set the `disabled` attribute.

Reply:
Thanks for your input here. I am going to contact the A11y testers and ask them if we can close this issue as Intended Behavior.